test/mlir/conversion/onnx_to_krnl/onnx_lowering_fuse.mlir

```diff
@@ -178,3 +178,94 @@ func.func @fuse_element_14(%arg0: tensor<5xf32>) -> tensor<*xf32> {
 // CHECK:           return [[RES_]] : memref<5xf32>
 // CHECK:         }
 }
+
+// -----
+
+func.func @fuse_element_15(%arg0: tensor<4x5xf32>, %arg1: tensor<?xf32>) -> tensor<*xf32> {
```

Comment:
Interesting: here the Add has a parameter that is not a constant but still is fused. Is that because it came from the args?

Reply:
yes.